core/vm/errors.go

```diff
@@ -6,14 +6,18 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
-type OutOfGasError struct{}
+var OutOfGasError = &OutOfGasErr{Message: "Out of gas"}
```

Comment:
For consistency with the other core errors, which makes use of struct type equality rather than variable equality. If we change this one we should change all, I think. @obscuren ?